codespell_lib/data/dictionary.txt

```diff
@@ -1733,6 +1733,8 @@ allocatoor->allocator
 allocatote->allocate
 allocatrd->allocated
 allocattion->allocation
+alloccate->allocate
+alloccated->allocated
```

Comment:
Perhaps add 
```
alloccates->allocates
alloccating->allocating
```
Not that there are many of these typos floating around, but we usually try to enter all verbal forms.

Reply:
Of course -- done!

Thanks for the quick review.